### PR TITLE
Build for ARM

### PR DESCRIPTION
**Pull request by ringsaturn**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: conda-forge.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+build_platform:
+  linux_aarch64: linux_aarch64
+  osx_arm64: osx_64
```
Comment (ringsaturn) [inline conda-forge.yml:3]: @synapticarbors would you like to look this change? I’m not sure whether could build for M1 Mac.

**Comment by synapticarbors** [comment]
It's probably best to not do this manually. There is a migration bot:
https://conda-forge.org/docs/maintainer/knowledge_base.html#apple-silicon-builds

I'll open the appropriate pull request.

**Comment by synapticarbors** [comment]
See: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5003

**Comment by ringsaturn** [comment]
Thanks for your information and I will close this.

**Pull request closed without merging**